### PR TITLE
Update for terraform 0 12

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

File: iam_users.tf [hunk 1]
```diff
@@ -129,9 +129,7 @@ data "aws_iam_policy_document" "iam_self_admin_doc" {
     ]
 
     resources = [
-      # The MFA ARN is identical to that of the user, except that the
-      # text "user" is replaced by "mfa"
-      "${replace(aws_iam_user.user.*.arn[count.index], "user", "mfa")}",
+      replace(aws_iam_user.user[count.index].arn, "user", "mfa"),
```
Comment (dav3r) [inline iam_users.tf:134]: There's a deleted comment here FYI.

**Review by dav3r — APPROVED** [review]
Looks good to me, though you probably want to add back that one comment (see my note)

File: iam_users.tf (hunk 1 above)
Comment (jsf9k) [reply iam_users.tf:134]: Thanks!  I missed that.

Comment (jsf9k) [reply iam_users.tf:134]: Comment added back in via commit 2e0a01dbdf3dac870bcbad7d068d8702dc7f42a2.

**Pull request merged**